src/main/java/com/openlattice/postgres/ResultSetAdapters.java

```diff
@@ -920,6 +922,27 @@ public static CollectionTemplateKey collectionTemplateKey( ResultSet rs ) throws
         return new CollectionTemplateKey( entitySetCollectionId, templateTypeId );
     }
 
+    public static OrganizationExternalDatabaseTable organizationAtlasTable( ResultSet rs ) throws SQLException {
+        UUID id = id(rs);
+        String name = name(rs);
+        String title = title(rs);
+        Optional<String> description = Optional.ofNullable( description( (rs) ) );
+        UUID organizationId = id(rs);
+
+        return new OrganizationExternalDatabaseTable( id, name, title, description, organizationId );
+    }
+
+    public static OrganizationExternalDatabaseColumn organizationAtlasColumn( ResultSet rs ) throws SQLException {
+        UUID id = id(rs);
+        String name = name(rs);
+        String title = title(rs);
+        Optional<String> description = Optional.ofNullable( description( (rs) ) );
+        UUID tableId = id(rs);
+        UUID organizationId = id(rs);
```

Comment:
same comment -- and `tableId` probably needs its own too

Reply:
Will do